src/gameboy/src/gameboy.cpp

```diff
@@ -4,8 +4,20 @@ namespace gameboy {
 
 Gameboy::Gameboy(std::string rom_path, bool debug_on) {
 	cartridge = std::make_unique<Cartridge>(rom_path);
+
+	/**
+	 *  Displaying the Meta Information of the Cartridge if it was a successful
+	 * load, else the TVP freezes
+	 */
+	if (cartridge->meta_data->is_logo_valid) {
+		cartridge->meta_data->display_meta();
+        Log::info("GameBoy Startup Successful!");
```

Comment:
Also, you can still display the metadata regardless of if the logo was valid or not.